php/libraries/User.class.inc

```diff
@@ -338,7 +338,7 @@ class User extends UserPermissions implements \LORIS\StudyEntities\AccessibleRes
     {
         $site_list = $this->userInfo['examiner'];
         unset($site_list['pending']);
-        natcasesort($site_list);
+        ksort($site_list);
```

Comment:
Why is this needed? It's a different sorting order..

Reply:
Site_list is a multiple dimension array. The natcasesort function only works with multiple dimension array in php7.4, not php8.  The ksort will do the same thing with natcasesort. only sorts the first level key.

Reply:
The sorting order is different. ksort sorts by key, while natcasesort sorts more human-like way for the front end (ie. the sorting goes "1", "2", .. "10" rather than "1, "10", "2" ..). But if that's not working with php8 and that's a trade-off we need to make, I guess having PHP8 tested is more important.